js-apps/meep-frontend/src/js/containers/meep-container.js

```diff
@@ -323,8 +323,12 @@ class MeepContainer extends Component {
     }, 2000);
   }
 
-  // Change & process scenario
   changeScenario(pageType, scenario) {
+    changeScenario(pageType, scenario, false);
+  }
+
+  // Change & process scenario
+  changeScenario(pageType, scenario, reInitVisView) {
```

Comment:
More recent linter does not accept functions with the same name. Suggest renaming one of the functions to avoid possible issues.

Reply:
updated accordingly